### ISSUE TITLE
Getting weird output on using a different correlation implementation

### ISSUE THREAD

**Issue by pranaypratyush** [NVlabs/PWC-Net#59]
Hi there, I tried using https://github.com/ClementPinard/Pytorch-Correlation-extension for the correlation with the arguments `(dilation_patch=1, kernel_size=1, stride=1, patch_size=(md * 2 + 1), padding=0)`.
But I keep getting weird output for the pre-trained models. I am on pytorch 1. Would that be of any concern? 
![download](https://user-images.githubusercontent.com/10587083/51100774-9fff8a80-17fd-11e9-9790-bc71309cac05.png)
![download 1](https://user-images.githubusercontent.com/10587083/51100778-a2fa7b00-17fd-11e9-8624-6ff1fd975494.png)

**Comment by luvegood** [comment]
You may follow the readme page for environment setup: https://github.com/NVlabs/PWC-Net/tree/master/PyTorch#installation I'm not able to help with issues with very diverse implementation details other than this one.

**Comment by xianshunw** [comment]
have you solved this problem? I also have such a problem.

**Comment by pranaypratyush** [comment]
I still haven't been able to solve this problem.

**Comment by lelelexxx** [comment]
@pranaypratyush  Is the value of optical flow extremely large? I have reimplement pwcnet using mxnet, and when i loaded the pytorch pre-trained weights into my mxnet model,  the EPE in MPI-Sintel and flyingchairs around 1000. Doubting about the different implement of correlation layer.
